src/features/vars-schema/schema.ts

```diff
@@ -31,11 +32,12 @@ export const schema: RequestHandler = (req, res) => {
 	const varsConfig = {
 		reservedNames: RESERVED_NAMES,
 		reservedNamespaces: RESERVED_NAMESPACES,
-		invalidRegex: INVALID_CHARACTER_REGEX.toString(),
+		invalidRegex: INVALID_ENV_VAR_REGEX.toString(),
 		whiteListedNames: ALLOWED_NAMES,
 		whiteListedNamespaces: ALLOWED_NAMESPACES,
 		blackListedNames: BLOCKED_NAMES,
 		configVarSchema,
+		configVarInvalidRegex: INVALID_CONFIG_VAR_REGEX.toString(),
```

Comment:
I'd keep them next to each other as it makes sense to group them imo:
```suggestion
		invalidRegex: INVALID_ENV_VAR_REGEX.toString(),
		configVarInvalidRegex: INVALID_CONFIG_VAR_REGEX.toString(),
		whiteListedNames: ALLOWED_NAMES,
		whiteListedNamespaces: ALLOWED_NAMESPACES,
		blackListedNames: BLOCKED_NAMES,
		configVarSchema,
```